doc/changes.md

```diff
@@ -1,4 +1,133 @@
 # Changelog
+## Release v0.4.0 (2021/06/20)
+
+[![Digital Object Identifier for PyGMT v0.4.0](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.4978645)
+
+### Highlights
+
+* 🎉 **Fourth minor release of PyGMT** 🎉
+* 10 new PyGMT functions and methods
+* New and improved gallery examples and tutorials
+* Drop GMT 6.1.1 support
```

Comment:
Add the "Tab Autocompletion" to Highlights?

Reply:
Added in https://github.com/GenericMappingTools/pygmt/pull/1341/commits/d06e0d469ff2e06f6be129c13478d51bbe4b45e6